polaris.shopify.com/src/pages/api/v0/tokens/index.tsx

```diff
@@ -14,7 +14,7 @@ const getGithubUrl = (file: string, isRaw: boolean) => {
 
 const html = `
 <!DOCTYPE>
-<html>
+<html lang="en">
```

Comment:
This HTML is actually for the scrappy landing page for the API.

To add the lang attriibute for all pages, you can add some shenanigans to next's config file, see #6475 .